content/applications/inventory_and_mrp/manufacturing/quality_control/instructions_check.rst

```diff
@@ -0,0 +1,85 @@
+Control Points`, and click :guilabel:`New`.
+
+On the quality check or |QCP| form, select :guilabel:`Instructions` from the :guilabel:`Type`
+drop-down field. If a single quality check is being created, enter the instructions for how to
+complete the check in the :guilabel:`Instructions` text entry field of the :guilabel:`Notes` tab at
+the bottom of the form. If a |QCP| is being configured, enter the instructions for how to complete
+the checks it creates in the text-entry field of the :guilabel:`Instructions` tab at the bottom of
+the form.
+
+Fill out the rest of the required fields for the quality check or |QCP|, as detailed in the
+documentation for :ref:`Quality checks <manufacturing/quality_control/quality-checks>` and
+:ref:`Quality Control Points <manufacturing/quality_control/quality-control-points>`. Finally, click
+:guilabel:`Save` to create the quality check or |QCP|.
+
+Process an Instructions quality check
+=====================================
+
+To process an Instructions quality check, select a manufacturing order or an inventory order
+(receipt, delivery, return, etc.). Manufacturing orders can be selected by navigating to
+:menuselection:`Manufacturing --> Operations --> Manufacturing Orders`, and clicking on an order.
+Inventory orders can be selected by navigating to :menuselection:`Inventory`, clicking the
+:guilabel:`# To Process` button on an operation card, and selecting an order.
+
+On the selected inventory or manufacturing order, a purple :guilabel:`Quality Checks` button appears
+at the top of the order if the order has quality checks that need to be completed. Click the button
+to open the :guilabel:`Quality Check` pop-up window, which shows all of the quality checks required
+for that order.
+
+.. image:: instructions_check/quality-check-pop-up.png
+   :align: center
+   :alt: The Quality Check pop-up window on a manufacturing or inventory order.
+
+To complete an Instructions quality check, follow the instructions detailed in the
+:guilabel:`Quality Check` pop-up window. Finally, click :guilabel:`Validate` to confirm that the
+check has been completed.
+
+Process a work order Instructions quality check
+===============================================
+
+When configuring a |QCP| that is triggered during manufacturing, a specific work order can also be
+specified in the :guilabel:`Work Order Operation` field on the |QCP| form. If a work order is
+specified, a quality alert is created for that specific work order, rather than the manufacturing
+order as a whole.
+
+Quality checks configured for work orders must be completed from the tablet view. To do so, begin by
+navigating to :menuselection:`Manufacturing --> Operations --> Manufacturing Orders`. Select a
+manufacturing order that includes a work order for which a quality check is required. Open the
+tablet view for that work order by clicking the :guilabel:`📱 (tablet)` tablet view button on the
```

Comment:
Wowow this mobile phone emoji is really cool! didn't now it's part of Unicode 2010 (: